message_tagging_service/__init__.py

```diff
@@ -1,6 +1,8 @@
 
 import logging
 
+from message_tagging_service.mts_config import mts_conf
+
 logging.basicConfig(
     format='%(asctime)s- %(name)s - %(levelname)s - %(message)s',
-    level=logging.DEBUG)
+    level=getattr(logging, mts_conf.log_level, 'INFO'))
```

Comment:
Is `getattr` needed here? It's defined in `BaseConfiguration` so any configuration class should technically have this property.

Reply:
Yeah, it is not needed. :) Thanks.